src/libs/DateUtils.js

```diff
@@ -25,15 +25,20 @@ Onyx.connect({
  * Moment object for the given timestamp
  *
  * @param {String} locale
+ * @param {Object} currentTimezone
```

Comment:
To denote it's an optional param:
```suggestion
 * @param {Object} [currentTimezone]
```